examples/psp22/README.md

```diff
@@ -47,22 +47,21 @@ pub mod my_psp20 {
 #[derive(Default, PSP22Storage, PSP22MetadataStorage)]
```

Comment:
Cannot comment the line, but the same above: 
according -> corresponding
Deriving these traits allow**s**